protostar/commands/declare_command.py

```diff
@@ -163,7 +164,7 @@ async def declare(
             response = await gateway_facade.declare(
                 compiled_contract_path=compiled_contract_path,
                 signer=signer,
-                account_address=account_address,
+                account_address=AccountAddress(int(account_address)),
```

Comment:
unnecessary casting to int

Reply:
resolving in favor of https://github.com/software-mansion/protostar/pull/1170#discussion_r1025372518